### ISSUE TITLE
onResize returns percentages instead of pixels when pixels are used

### ISSUE THREAD

**Issue by hipstersmoothie** [bvaughn/react-resizable-panels#180]
https://codesandbox.io/s/react-resizable-panels-forked-z3hp8h?file=/src/App.js

**Comment by simpslandyy** [comment]
I'm also experiencing this problem. Is there an estimated time this will get addressed?

**Comment by bvaughn** [comment]
> Is there an estimated time this will get addressed?

No. This is not a for-profit package so there's no SLA 😉 

It should be fixed once I have time to finish #182

**Comment by simpslandyy** [comment]
> > Is there an estimated time this will get addressed?
> 
> No. This is not a for-profit package so there's no SLA 😉
> 
> It should be fixed once I have time to finish #182

No worries thank you! 👍🏽

**Comment by bvaughn** [comment]
This has been resolved by #194 which I plan to publish later this afternoon. The `onResize` method will now receive the following:

```ts
type MixedSizes = {
  sizePercentage: number;
  sizePixels: number;
}

onResize(
  current: MixedSize,
  prev: MixedSize)
);
```
